content/en/blog/2020/on-demand-egress-gateways/index.md

```diff
@@ -0,0 +1,500 @@
+
+{{< image width="75%" ratio="45.34%"
+    link="./on-demand-egress-gateway-overview.svg"
+    alt="Overview multiple Egress Gateways"
+    caption="Overview multiple Egress Gateways"
+    >}}
+
+With this model, one `Egress Gateway` is in charge of only one upstream service.
+
+Although the `IstioOperator` spec allows you to deploy multiple `Egress Gateways`, the manifest can become unmanageable.
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+[...]
+spec:
+    egressGateways:
+    - name: egressgateway-1
+      enabled: true
+    - name: egressgateway-2
+      enabled: true
+    [egressgateway-3, egressgateway-4 ,...]
+    - name: egressgateway-N
+      enabled: true
+[...]
+{{< /text >}}
+
+As a benefit of decoupling Egress Getaways from IstioOperator manifest, you have enabled the possibility of setting up **custom readiness probes** to have both services (gateway and upstream service) aligned.
+
+As well, you can apply the sidecar pattern to inject, for example, OPA into the pod to perform authorization with complex rules ([OPA envoy plugin](https://github.com/open-policy-agent/opa-envoy-plugin)).
+
+{{< image width="75%" ratio="45.34%"
+    link="./on-demand-egress-gateway-authz.svg"
+    alt="Authorization with OPA and `healthcheck` to external service"
+    caption="Authorization with OPA and `healthcheck` to external"
+    >}}
+
+At this point, you might be convinced that this is the right path to go.
+
+## Solution
+
+There are several ways to achieve this task:
+
+- Create multiple `IstioOperator` (simple solution)
+- Create an operator (recommended for a second iteration in the development process)
+
+At **PAN-NET**, we have created our own operator with the [Operator SDK - Ansible type](https://sdk.operatorframework.io/docs/building-operators/).
+
+However, you want to start simple. Thus, better you start with the `IstioOperator` option.
+
+{{< quote >}}
+Yes! `Istio 1.8.0` introduced the possibility to have fine-grained control over the objects that `IstioOperator` deploys. This gives you the opportunity to patch them as you wish. Exactly what you need to deploy on demand `Egress Gateways`.
+{{< /quote >}}
+
+In the following section you will  deploy an `Egress Gateway` to connect to an external service: `httpbin` ([https://httpbin.org/](https://httpbin.org/))
+
+At the end, you will have:
+
+{{< image width="75%" ratio="45.34%"
+    link="./on-demand-egress-gateway-communicatin.svg"
+    alt="Communication"
+    caption="Communication"
+    >}}
+
+## Hands on
+
+### Prerequisites
+
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) (`kubernetes in docker`. Perfect for local development)
+- [Istioctl](/docs/setup/getting-started/#download)
+
+#### Kind
+
+{{< warning >}}
+If you use `kind`, do not forget to set up `service-account-issuer` and `service-account-signing-key-file` as described below. Otherwise, `Istio` installation might complain.
```

Comment:
```suggestion
If you use `kind`, do not forget to set up `service-account-issuer` and `service-account-signing-key-file` as described below. Otherwise, Istio may not install correctly.
```